mythril/analysis/modules/external_calls.py

```diff
@@ -91,17 +91,16 @@ def _analyze_state(self, state: GlobalState):
                 state, constraints + state.world_state.constraints
             )
 
+            for tx in state.world_state.transaction_sequence:
+                if not isinstance(tx, ContractCreationTransaction):
+                    constraints.append(tx.caller == ACTORS.attacker)
```

Comment:
Should this not be something that appends to new_constraints?

Reply:
I don't think it should be part of the constraints reported if the next sat soliving fails